### ISSUE TITLE
whitedb doesn't release shared memory on Windows platform

### ISSUE THREAD

**Issue by priitj** [priitj/whitedb#30]
Applications may need to release some of the shared memory while running.

Currently this is not implemented via whitedb API.

**Comment by rohanbabu** [comment]
Does this mean that the size of the DB will only grow and that the wg_delete_record(void* db, void *rec) function does not reduce the size of the DB even if it successful?

**Comment by priitj** [comment]
Nope, this issue is about wg_detach_database()/wg_delete_database().

Records are allocated internally by whitedb, creating or deleting them does not change the size of the shared memory area.
